vnext/Microsoft.ReactNative.SharedManaged/JSValueReaderGenerator.cs

```diff
@@ -211,13 +214,33 @@ private static Delegate GenerateReadValueForEnum(Type valueType)
         : null;
     }
 
+    public static void ReadClass<TClass>(IJSValueReader reader, out TClass value) where TClass : new()
+    {
+      // .Net Native seems to have a bug that does not allow to have 'out' or 'ref' parameters.
```

Comment:
>bug [](start = 37, length = 3)

(nit) would be nice to contact .net team with this issue :)

Reply:
I did. No response.

---
In reply to: [414051939](https://github.com/microsoft/react-native-windows/pull/4696#discussion_r414051939) [](ancestors = 414051939)

Reply:
@vmoroz could you file a bug in the dotnet runtime project?
https://github.com/dotnet/runtime